torch/_functorch/_aot_autograd/autograd_cache.py

```diff
@@ -63,11 +63,36 @@ def check_node_safe(node: Node):
     """
     Checks that the node only uses supported operators. We are starting with very
     conservative cacheability constraints, and incrementally adding more support as we expand.
+
+    [Note: AOTAutograd Cacheability checks]
+    - Our cache key is computed from the FX graph produced by Dynamo and the input example values
+    - A node is "safe" if the same cache key results in a compiled artifact that has the same behavior
+        (i.e, the set of inputs that go into our cache key is sufficient to distinguish its behavior)
+
+    To accomplish this safety check, we consider the following functions to be safe:
+        - Public functions under modules torch, torch.functional, and torch.nn.functional: these are
```

Comment:
I think the `"under torch"` part is out of date? (given Richard's worry about "stuff people put under `torch.foo` today that might be unsafe to cache, like HOPs).

I agree we are just kind of punting the problem - although it would probably be useful to see the cache start to miss by running it more (maybe even on the dynamo CI shard?), and see what ops actually show up in `torch.foo` that fail the cache check today?)